packages/app/src/components/search/SearchPage.tsx

```diff
@@ -45,7 +45,7 @@ import BuildIcon from '@material-ui/icons/Build';
 
 const useStyles = makeStyles((theme: Theme) => ({
   bar: {
-    padding: theme.spacing(1, 0),
+    padding: theme.spacing(1),
```

Comment:
this doesn't seem to be used anymore?

Reply:
Not sure what that means, I removed the `0` to have padding all around.

Reply:
Oh sorry, my comment was not so clear. I meant that `bar` is not used from classes anymore as it was taken away here https://github.com/backstage/backstage/pull/16647/files#diff-d6acc6818a6e603a0914a3389bc230a41f2af3c51d7874284baafe37bbadccecL73 so I was thinking that it might be possible to remove this!

Reply:
Ah the `bar` class, will remove if no other reference in use.